README.md

```diff
@@ -12,10 +12,25 @@ Consisting of a data availability solution and a decentralized network of nodes
 rollups can leverage the Espresso Sequencer to give developers and end users fast confirmations, low (and fair) fees,
 and robust infrastructure.
 
-[Documentation](https://docs.espressosys.com/sequencer/espresso-sequencer-architecture/readme)
+[Official Documentation](https://docs.espressosys.com/sequencer/espresso-sequencer-architecture/readme)
+
+### Architecture
+
+This diagram below depicts a simplified view of the current architecture of the Espresso Sequencer.
```

Comment:
Do we have a public mempool the user could send her transaction to?

Reply:
I tried to add a mempool but had a hard time drawing and it quickly seems to go into hotshot internals. So I decided to add more arrows to indicate that transactions are propagated through the network. Is that fine?

I didn't draw the user submitting directly to the builder. Do you think it's worthwhile? I think it's fine to just add the "most common" case.

Reply:
4f98ca57e8840ceb2965dba2e6c11c9211f42c74

Reply:
Ok, I see. That is fine to me.